src/bindgen/ir/constant.rs

```diff
@@ -183,7 +207,26 @@ impl Literal {
                 }
                 _ => Err(format!("Unsupported Unary expression. {:?}", *op)),
             },
-            _ => Err(format!("Unsupported literal expression. {:?}", *expr)),
+
+            // Match identifiers, like `5 << SHIFT`
+            syn::Expr::Path(syn::ExprPath {
+                path: syn::Path { ref segments, .. },
+                ..
+            }) => {
+                // Handle only the simplest identifiers and error for anything else.
+                // NOTE: Should this use `ir::Path` instead?
+                if segments.len() == 1 {
+                    Ok(Literal::Expr(format!("{}", segments.last().unwrap().ident)))
```

Comment:
This should be a different kind of `Literal`, so that it can work correctly with `export.prefix` and such configuration options. `rename_for_config` should be called on it.

Please add a test for this to the `prefix` tests.

Reply:
> This should be a different kind of Literal

@emilio  Just to be clear, did you mean I should add a variant to enum `ir::constant::Literal` to handle names instead of reusing Expr? That would indeed be better. What do you think we should call it? Name/Reference/Segment/Path ? 

Makes sense about renaming for config, will do that and add a test :+1:

Reply:
Yeah. `Literal::Path` would make sense to me, wdyt?

Reply:
Literal::Path(String) sounds good!